app/containers/App/index.js

```diff
@@ -49,7 +49,7 @@ export default function App() {
         <NavBar />
         <Switch>
           <Route exact path="/" component={Landing} />
-          <Route path="/:companyName" component={HomePage} />
+          <Route path="/company/:companyId/table/:tableNumber" component={Table} />
```

Comment:
Is the routes /company and /table necessaries? We could display just the company name and the table number, 
like: /companyName/table-1. 
I think it's not a good practice showing  id's in the URL. Anyway, the url isn't displayed in the smartphone. So, it inst't a big problem to worry about,